cloudbuild.yaml

```diff
@@ -4,4 +4,6 @@ steps:
 - name: 'gcr.io/config-validator/make'
   args: ['check_sample_files']
 - name: 'gcr.io/config-validator/make'
-  args: ['check_format']
\ No newline at end of file
+  args: ['check_format']
+- name: 'gcr.io/config-validator/policy-tool'
+  args: ['lint', '--policies', './policies', '--libs', './lib']
```

Comment:
Also add "--policies", "./samples" to the args

Reply:
Fixed- thanks!